src/MICore/LaunchOptions.cs

```diff
@@ -203,6 +227,11 @@ static internal LocalLaunchOptions CreateFromXml(Xml.LaunchOptions.LocalLaunchOp
             options.InitializeCommonOptions(source);
             options.InitializeServerOptions(source);
 
+            if (LaunchCompleteCommand.CoreDump == options.LaunchCompleteCommand)
```

Comment:
same as above. LaunchCompleteCommand doesn't seem like the right place for this state